app/main.py

```diff
@@ -1,6 +1,25 @@
+import os
+import requests
+
+API_URL = "http://api.weatherapi.com/v1/current.json"
+CITY = "Paris"
```

Comment:
Consider making `API_URL` and `CITY` configurable through environment variables or command-line arguments. This will make the script more flexible and reusable for different cities or API endpoints.